web/src/AccountMenu.tsx

```diff
@@ -166,3 +167,33 @@ export function AccountMenuHeader(props: AccountMenuProps) {
     </AccountMenuHeaderRoot>
   )
 }
+
+type AccountMenuDialogProps = {
+  open: boolean
+  anchorEl: HTMLElement | null
+  onClose: () => void
+} & AccountMenuProps
+
+export function AccountMenuDialog(props: AccountMenuDialogProps) {
+  const { open, anchorEl, onClose, ...accountMenuProps } = props
+  const features = useFeatures()
+
+  const hideAccountDialog = features.isEnabled(Flag.OfflineSnapshotCreation)
+  if (hideAccountDialog) {
+    return null
```

Comment:
this might be a silly question...but shouldn't it be impossible to hit this codepath? because disabling the dialog happens further upstream.

Reply:
not a silly question! i'm being extra-defensive in this code path and mostly added this case due to my local testing. you can open the account dialog, enable the feature flag, and the dialog will stick around even though the menu item is gone.